app/representers/api/msg/image_representer.rb

```diff
@@ -4,6 +4,12 @@
 # Representer for Announce messaging
 #
 class Api::Msg::ImageRepresenter < Api::BaseRepresenter
+  include NestedImage
+
+  def self_url(represented)
```

Comment:
The announce-representer doesn't actually need this.  But I guess it can't hurt for it to know the CMS-url for the image as well.